cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -2093,6 +2093,16 @@ spec:
                         type: string
                       startTime:
                         type: string
+            quotas:
+              properties:
+                enabled:
+                  type: boolean
+                maxBytes:
+                  type: integer
+                  minimum: 0
```

Comment:
The `minimum` needs to be set in the other locations where the schema was copied as well.

Reply:
Fixed.